.github/workflows/build-test-deploy.yml

```diff
@@ -270,10 +270,8 @@ jobs:
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_NEW }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_NEW }}
```

Comment:
nit: can we use a better naming convention for these keys instead of `_NEW`? Just so we can support potential future upgrades without devolving into `_NEW_NEW...`.
Perhaps `_OIDC` or a version number?

Reply:
After this change we won't store long term AWS credentials.
https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-cloud-providers#overview